### ISSUE TITLE
Perform GridSearch?

### ISSUE THREAD

**Issue by mcormick123** [amaiya/ktrain#494]
Hi, i'm fine tuning mBERT for sentiment analysis, i want to fine tune using batch sizes [8, 16] and learning rates [2e-5, 5e-5], can i perform gridsearch on that in ktrain? or should i just manually change it and rerun the entire program? I think it will affect the results since train and test sets aren't the same anymore. Thank you!

**Comment by amaiya** [comment]
You can either do it manually or use a tool like Weights & Biases with **ktrain** to tune hyperparameters.  Since changes to batch size and learning rate do not require you to reload/re-process the data, the train test split should not change.  However, if you do need to re-process and re-split the training set into a validation set, you can set the `random_state` parameter in any of the `texts_from*` functions for reproducibility:
```
    random_state(int):      If integer is supplied, train/test split is reproducible.
                            If None, train/test split will be random
```
